### PR TITLE
initialize project with components

### PR DESCRIPTION
**Pull request by HanaSabih**
-  Create React application.
-  Add Books and Categories Pages.
-  Add required components.
-  Add some style.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// import Container from 'react-bootstrap/Container';
```
Comment (zieeco) [inline src/components/Book.js]: - Please, kindly remove this commented code to make your codebase clean.

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </Card>
+    // <Container>
+
+  // <article className="one-book">
+  //   <p className="book-title">{title}</p>
+  //   <p className="book-author">{author}</p>
+  //   <button type="button" className="add-btn">delete</button>
+  // </article>
+  // </Container>
+  );
```
Comment (zieeco) [inline src/components/Book.js]: - Please, kindly remove this commented code to make your codebase clean.

File: src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+// import { Outlet, Link } from 'react-router-dom';
```
Comment (zieeco) [inline src/components/Navbar.js]: - Please, kindly remove this commented code to make your codebase clean.

File: src/components/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+//           </Nav>
+
+//         </Navbar.Collapse>
+//       </Container>
+//     </Navbar>
+//     <Outlet />
+//   </>
+// );
+
+// export default Navs;
```
Comment (zieeco) [inline src/components/Navbar.js]: - Please, kindly remove this commented code to make your codebase clean.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @HanaSabih,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @HanaSabih , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
Gitflow :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**